added prometheus sample for keda2

kcfigaro opened the pull request:
added prometheus sample for KEDA2
[Autoscale Kubernetes pods based on ingress request — Prometheus, KEDA, and K6](https://blog.cloudacode.com/how-to-autoscale-kubernetes-pods-based-on-ingress-request-prometheus-keda-and-k6-84ae4250a9f3)

<!-- Thank you for contributing!

     Read more about how you can contribute in our contribution guide:
     https://github.com/kedacore/keda/blob/master/CONTRIBUTING.md
-->

_Provide a description of what has been changed_

### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)
- [x] A PR is opened to update the documentation on [our docs repo](https://github.com/kedacore/keda-docs)

Fixes #

kcfigaro committed:
added prometheus sample for KEDA2

[Autoscale Kubernetes Pods based on ingress request — prometheus, KEDA2, and k6](https://blog.cloudacode.com/how-to-autoscale-kubernetes-pods-based-on-ingress-request-prometheus-keda-and-k6-84ae4250a9f3)

Signed-off-by: KC Chang <kc@cloudacode.com>
Signed-off-by: kcfigaro <kc@cloudacode.com>

zroubalik reviewed (approved):
Looking good!

Could you please update the blogpost (if that's possible) to make sure KEDA is spelled with uppercase, so no Keda or keda. Thanks!

Also, could you please fix DCO?

To add your Signed-off-by line to every commit in this branch:

    Ensure you have a local copy of your branch by [checking out the pull request locally via command line](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/checking-out-pull-requests-locally).
    In your local branch, run: git rebase HEAD~2 --signoff
    Force push your changes to overwrite the branch: git push --force-with-lease origin main

kcfigaro committed:
updated the prometheus sample title name

Signed-off-by: kcfigaro <kc@cloudacode.com>

kcfigaro commented:
Thanks, @zroubalik! 
I added the signed-off-by in my commit accordingly and also updated the KEDA spell in my blog.

Could you kindly review it again?

zroubalik commented:
@kcfigaro thanks. Thanks for the addition, it's a great stuff!
